ui/raidboss/data/07-dt/alliance/jeuno-first-walk.ts

```diff
@@ -0,0 +1,792 @@
+      type: 'StartsUsing',
+      netRegex: { id: '9BF2', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 7,
+      response: Responses.stackMarker(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Sharp Spike Collect',
+      type: 'HeadMarker',
+      netRegex: { id: '0156', capture: true },
+      run: (data, matches) => data.spikeTargets.push(matches.target),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Sharp Spike Call',
+      type: 'StartsUsing',
+      netRegex: { id: '9F97', source: 'Fafnir The Forgotten', capture: false },
+      delaySeconds: 0.5,
+      durationSeconds: 6.5,
+      alertText: (data, _matches, output) => {
+        if (data.spikeTargets?.includes(data.me))
+          return output.cleaveOnYou!();
+        return output.avoidCleave!();
+      },
+      run: (data) => {
+        // Dragon Breath is also deleted here because Sharp Spike
+        // consistently follows Dragon Breath,
+        // while Touchdown does not.
+        data.spikeTargets = [];
+        delete data.dragonBreathFacingNumber;
+      },
+      outputStrings: {
+        cleaveOnYou: Outputs.tankCleaveOnYou,
+        avoidCleave: Outputs.avoidTankCleaves,
+      },
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Horrid Roar Spread',
+      type: 'HeadMarker',
+      netRegex: { id: '01F3', capture: true },
+      condition: Conditions.targetIsYou(),
+      response: Responses.spread(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Absolute Terror',
+      type: 'StartsUsing',
+      netRegex: { id: '9F8D', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 5,
+      response: Responses.goSides(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Winged Terror',
+      type: 'StartsUsing',
+      netRegex: { id: '9F8F', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 5,
+      response: Responses.goMiddle(),
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Hurricane Wing Outer Ring',
+      type: 'StartsUsing',
+      netRegex: { id: '9F78', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.outerFirst!(),
+      outputStrings: {
+        outerFirst: {
+          en: 'Rings out to in',
+        },
+      },
+    },
+    {
+      id: 'Jeuno First Walk Fafnir Hurricane Wing Inner Ring',
+      type: 'StartsUsing',
+      netRegex: { id: '9F7D', source: 'Fafnir The Forgotten', capture: false },
+      durationSeconds: 5,
+      infoText: (_data, _matches, output) => output.outerFirst!(),
+      outputStrings: {
+        outerFirst: {
+          en: 'Rings in to out',
+        },
+      },
+    },
+    {
+      id: 'Jeuno First Walk Angels CloudSplitter Collect',
+      type: 'HeadMarker',
+      netRegex: { id: '01D0', capture: true },
+      run: (data, matches) => data.splitterTargets.push(matches.target),
+    },
+    {
+      id: 'Jeuno First Walk Angels CloudSplitter Call',
```

Comment:
This trigger didn't fire correctly when I tested, could be a race condition.

Reply:
It looks like this might be a situation where the castbar source doesn't always align with the invisible on-target cast sources. We could instead  use `A076`, now that I look at it, since even though it's a shorter cast time it starts simultaneously with `A077`, and is consistently used by MR.